.github/workflows/codeql.yml

```diff
@@ -57,27 +65,10 @@ jobs:
         # Run extended queries including queries using machine learning
         queries: security-extended
 
-    - name: Set up Go
-      if: matrix.language == 'go'
-      uses: actions/setup-go@v5
-      with:
-        go-version-file: go.mod
```

Comment:
No longer needed. Specifying the go language in the `github/codeql-action/init@v3` step now gets the correct version, this didn't previously happen.